contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/AvroStoreBuilderMapper.java

```diff
@@ -112,11 +115,33 @@ public void map(GenericData.Record record,
                     AvroCollector<Pair<ByteBuffer, ByteBuffer>> collector,
                     Reporter reporter) throws IOException {
 
-        byte[] keyBytes = keySerializer.toBytes(record.get(keyField));
-        byte[] valBytes = valueSerializer.toBytes(record.get(valField));
+        byte[] keyBytes = null, valBytes = null;
+        Object keyRecord = null, valRecord = null;
```

Comment:
Voldemort predates the LI style guide and generally doesn't follow it, but sure. I can split those lines.